### PR TITLE
docs - add book info, content updates required to get the docs to build

### PR DESCRIPTION
**Pull request by lisakowen**
this PR includes the book configuration and content updates required to build the open source pxf docs from the content added to this repository in https://github.com/greenplum-db/pxf/pull/374.  in this PR:
- add the book/ contents - the majority of the book was copied from the https://github.com/greenplum-db/gpdb/tree/master/gpdb-doc/book directory
- added a README.md describing how to build the open source pxf docs
- doc content files - replaced peer cross-references with full greenplum docs URL

note:  the pxf book uses bookbinder to build the pxf docs, just as the greenplum docs do.  bookbinder is a ruby build environment.  i plan to enhance the README, and possibly provide a Docker file that one could use to build the docs on a linux system in a separate PR.

doc review site link:  http://docs-lisa-pxf-oss.cfapps.io/pxf/WIP/overview_pxf.html

### CODE REVIEW

File: docs/content/sdk/deploy_conn.html.md.erb [hunk 1]
```diff
@@ -47,7 +47,7 @@ The administrator must also install any third-party commands or other components
 
 To verify that you deployed a connector successfully, you create a Greenplum Database external table and invoke `SELECT` and/or `INSERT` commands on the table to test read and write operations on the external data source.
 
-The Greenplum Database end user accesses an external data source by invoking a `CREATE EXTERNAL TABLE` command specifying the `pxf` protocol. (Refer to the [CREATE EXTERNAL TABLE](../../ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) reference page for more about this Greenplum Database command.)
+The Greenplum Database end user accesses an external data source by invoking a `CREATE EXTERNAL TABLE` command specifying the `pxf` protocol. (Refer to the [CREATE EXTERNAL TABLE](https://gpdb.docs.pivotal.io/latest/ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) reference page in the Greenplum documentaion for more about this command.)
```
Comment (frankgh) [inline docs/content/sdk/deploy_conn.html.md.erb:50]:
```suggestion
The Greenplum Database end user accesses an external data source by invoking a `CREATE EXTERNAL TABLE` command specifying the `pxf` protocol. (Refer to the [CREATE EXTERNAL TABLE](https://gpdb.docs.pivotal.io/latest/ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) reference page in the Greenplum documentation for more about this command.)
```

File: docs/content/intro_pxf.html.md.erb [hunk 2]
```diff
@@ -25,8 +25,7 @@ PXF bundles all of the Hadoop JAR files on which it depends, and supports the fo
 
 | PXF Version | Hadoop Version | Hive Server Version | HBase Server Version |
 |-------------|----------------|---------------------|-------------|
-| 5.10, 5.11, 5.12 | 2.x, 3.1+ | 1.x, 2.x, 3.1+ | 1.3.2 |
-| 5.9 | 2.x, 3.1+ | 1.x, 2.x, 3.1+ | 1.3.2 |
+| 5.9, 5.10, 5.11, 5.12 | 2.x, 3.1+ | 1.x, 2.x, 3.1+ | 1.3.2 |
```
Comment (frankgh) [inline docs/content/intro_pxf.html.md.erb:28]: should we say 5.9+ here?

File: docs/content/hive_pxf.html.md.erb [hunk 3]
```diff
@@ -189,7 +189,7 @@ LOCATION ('pxf://<hive-db-name>.<hive-table-name>
 FORMAT 'CUSTOM|TEXT' (FORMATTER='pxfwritable_import' | delimiter='<delim>')
 ```
 
-Hive connector-specific keywords and values used in the [CREATE EXTERNAL TABLE](../ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) call are described below.
+Hive connector-specific keywords and values used in the Greenplum Database [CREATE EXTERNAL TABLE](https://gpdb.docs.pivotal.io/latest/ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) call are described below.
```
Comment (frankgh) [inline docs/content/hive_pxf.html.md.erb:192]: not sure why this line has `Greenplum Database [CREATE EXTERNAL TABLE]..` but in other places Greenplum Database was not added. Is this on purpose?

File: docs/content/hdfs_text.html.md.erb [hunk 4]
```diff
@@ -136,7 +136,7 @@ LOCATION ('pxf://<path-to-hdfs-file>?PROFILE=hdfs:text:multi[&SERVER=<server_nam
 FORMAT '[TEXT|CSV]' (delimiter[=|<space>][E]'<delim_value>');
 ```
 
-The specific keywords and values used in the [CREATE EXTERNAL TABLE](../ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) command are described in the table below.
+The specific keywords and values used in the [CREATE EXTERNAL TABLE](https://gpdb.docs.pivotal.io/latest/ref_guide/sql_commands/CREATE_EXTERNAL_TABLE.html) command are described in the table below.
```
Comment (frankgh) [inline docs/content/hdfs_text.html.md.erb:139]: this is missing a `Greenplum Database` before `[CREATE EXTERNAL TABLE]`. Is this on purpose?

File: docs/content/hive_pxf.html.md.erb (hunk 3 above)
Comment (lisakowen) [reply docs/content/hive_pxf.html.md.erb:192]: i tried to a Greenplum Database to first xref to CREATE EXTERNAL TABLE.  previously, pxf and greenplum were in the same doc set.  now that pxf has its own docset, i wanted to make it obvious that the link would take them to greenplum.

**Pull request merged**